packages/sit-onyx/src/components/OnyxInfoTooltip/OnyxInfoTooltip.vue

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" setup>
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+import circleInformation from "@sit-onyx/icons/circle-information.svg?raw";
+import OnyxTooltip from "../OnyxTooltip/OnyxTooltip.vue";
+import type { OnyxInfoTooltipProps } from "./types";
+import { injectI18n } from "../../i18n";
+
+const props = defineProps<OnyxInfoTooltipProps>();
+const { t } = injectI18n();
+</script>
+
+<template>
+  <OnyxTooltip
+    v-if="props.text"
```

Comment:
We should make the text prop required instead, otherwise people might be confused why the component is not shown although they use it and think its a bug in the component

Reply:
Since I'm picking the type from OnyxTooltip, how do I make it required only here?

![image](https://github.com/SchwarzIT/onyx/assets/60856270/a1892c0a-c0b8-4a07-a09e-e470194255ec)

Reply:
```ts
NonNullable<Pick<OnyxTooltipProps, "text">> & Pick<OnyxTooltipProps, "position">
```

So everything you put in the left side will be required and everything on the right will be picked as it is